CFP/Speakers Committee RunBook (#10)
devopsdays-austin/runbooks: speakers-cfp_runbook.md

# Speakers/CFP

Scope:

This document is the expectations and scope of what the CFP team should be do for DevOpsDays Austin.
Most of these are points are expected, but to be completely clear this is minimal expectations.

## Role description

The CFP commitee (or team) is the group of individuals that brings the agenda together for the event. Without
the CFP commitee the event would have a challanging time bringing people together to discuss and engage
in our DevOps focuesed event. Many of the talks that we help curate help shape the converstations and
Open Spaces for DoD ATX, and takes commitment throughout the year to understand what would bring the
best converastions to Austin.

### General Tasks

Can you name typical tasks? For example:

* Write a compelling CFP (the CFP committee)
* Engage with the Open Source communities that we are privliged to be apart of
* Vote and read all the CFPs that are proposed

> * Attempt to be at every meeting for the smaller and larger team, things can change quickly and CFPs can be effected

Reviewer: ```suggestion
* Attempt to be at every meeting for the smaller and larger team (things can change quickly and CFPs can be affected)
```
I feel like this is a wider responsibility and not a task. Core will look into a wider responsibility list like this.

> * Recurit or indentify possible Key Note speakers

Reviewer: ```suggestion
* Recruit or identify possible keynote speakers
```

* Work closely with the Sponsorship and Core team to make sure that the ratio of talks to workshops to open spaces is reasonable
* Comment or filter out "sales pitches" actively through the Open CFP process
* Administer the CFP software for DoD ATX
* Timely answer any and all questions for that comes in from possible or future speakers

> * Attend and Run the Office Hours, pre and post CFP for new and experianced speakers

Reviewer: ```suggestion
* Attend and Run the Office Hours, pre and post CFP for new and experienced speakers
* Finalize the speaker list
* Send out acceptances
* Schedule the speakers in coordination with core
* MC the stages day of in coordination with core
```

## Important Dates

### In the beginning

> * Copy the previous years CFP, make any changes for the new theme from Core

Reviewer: ```suggestion
* Copy the previous year's CFP and make any changes for the new theme from Core
```

* Verify that the CFP software is ready from Global Core

> * Start discussing with the Venue team to verify options if the venue has changed for workshops and breakout spaces

Reviewer: I'd argue this needs to happen before the CFP is written, yes?

Author: My theory is that our CFP is more or less "static" for the foreseeable future. We update it with the theme and once over with anything that is obvious from last year than doesn't need to be here this year.

The amount of sessions is only really changed if we decide to axe (or add) something and I don't see us removing the three types any time soon.

### 7-5 Months Out

> * Prepare the CFP and publicly annouce the CFP were you can. (the specific day will require feedback from Core)

Reviewer: ```suggestion
* Prepare the CFP
* Announce the CFP where you can. Coordinate with Core and Marketing.
```

* Start discussing with possible people for Key Note slots or conversations they believe they would like to share.
* Recruit speakers!
* Recruit speakers!
* Recruit speakers!

### 4 months Out

> * Start a cadance of the Office Hours (once a week or some reasonable process)

Reviewer: ```suggestion
* Start a cadence of the Office Hours (once a week or some reasonable process)
```

### 3-2 months Out

* Close down the CFP
* Start to vote and recruit others from either outside or inside the Org team to help vote on the speakers

> * Send out the speaker acceptences

Reviewer: ```suggestion
* Send out the speaker acceptances
```

> * Continue a cadance of the Office Hours (once a week or some reasonable process)

Reviewer: ```suggestion
* Continue a cadence of the Office Hours (once a week or some reasonable process)
```

> ### 1 Month Out

Reviewer: Schedule?

Author: Good question, i'll have to chew on this one.

Reviewer 2: what about
* Work in partnership with the core team to build out the schedule for the event.
* Maintain communication with the speakers.
* Provide feedback on alignment with the theme and focus on attendee experience.

* Verify the speakers and nothing has changed last minute

> * Continue a cadance of the Office Hours (once a week or some reasonable process)

Reviewer: ```suggestion
* Continue a cadence of the Office Hours (once a week or some reasonable process)
```

### The Day Of

* Be prepared to MC the breakout rooms
* Be there for the speakers and what they may need
* Remember you are the representive of DoD ATX to many of these speakers, be there and friendly!

## Reminders or Common Mistakes

* Spend "real time" with the CFP software before opening the CFP. It can be tricky

> * You DONT have to "promote" a someone in via the CFP, you can recruit your Key Note(s) speaker(s)

Reviewer: ```suggestion
* You DON'T have to "promote" someone who submitted via the CFP to a keynote slot. Recruit your keynote speaker(s) as needed.
```

> * Gain the muscule to find "sales pitches" abstracts, but don't beat yourself up if one slips through

Reviewer: ```suggestion
* Gain the muscle to find "sales pitches" abstracts, but don't beat yourself up if one slips through
```

### 7-5 Months Out

* Prepare the CFP
* Announce the CFP where you can. Coordinate with Core and Marketing.
* Start discussing with possible people for Key Note slots or conversations they believe they would like to share.
* Recruit speakers!
* Recruit speakers!
* Recruit speakers!

### 4 months Out

* Start a cadence of the Office Hours (once a week or some reasonable process)

### 3-2 months Out

* Close down the CFP
* Start to vote and recruit others from either outside or inside the Org team to help vote on the speakers
* Send out the speaker acceptances
* Continue a cadence of the Office Hours (once a week or some reasonable process)

### 1 Month Out

* Verify the speakers and nothing has changed last minute
* Continue a cadence of the Office Hours (once a week or some reasonable process)

### The Day Of

* Be prepared to MC the breakout rooms
* Be there for the speakers and what they may need
* Remember you are the representive of DoD ATX to many of these speakers, be there and friendly!

## Reminders or Common Mistakes

* Spend "real time" with the CFP software before opening the CFP. It can be tricky

> * You DON'T have to "promote" someone who submitted via the CFP to a keynote slot. Recruit your keynote speaker(s) as needed.

Reviewer 2: really keynotes should be a core responsibility, emphasis on feedback rather than selection/recruitment here?
